acra-core/src/main/AndroidManifest.xml

```diff
@@ -25,6 +26,7 @@
             android:process=":acra" />
 
         <provider
+            tools:replace="android:authorities"
```

Comment:
What's this for?

Reply:
It allows fields to be replaced when merging manifests https://developer.android.com/studio/build/manifest-merge
I think it may have been required because ${applicationId} gets substituted the first time it is included. So if you have an AAR that depends on ACRA then it gets substituted there, and doesn't when your AAR is then included in an APK. Leaving you with invalid authorities.